frontend/editor/index.html

```diff
@@ -7,78 +7,104 @@
 		</form>
-		</p>
 	</div>
-	<hr>
-	<button id="lock">Lock</button>
-	<button id="unlock">Unlock</button>
-	<button id="pull">Pull</button>
-	<button id="push">Push</button>
-	<span id="controls_error" class="error"></span>
-	<hr>
-	<h2>Schedule Type information</h2>
-	<select id="select_schedule">
-	</select>
-	<button id="add_schedule">+</button>
-	<button id="copy_schedule">copy</button>
-	<button id="remove_schedule">-</button>
-	<br>
-	<label for="schedule_friendly_name">Friendly Name</label>
-	<input id="schedule_friendly_name">
-	<br>
-	<label for="schedule_regex">Schedule Regex</label>
-	<input id="schedule_regex">
-	<hr>
-	<h2>Period Information</h2>
-	<select id="select_period">
-	</select>
-	<button id="add_period">+</button>
-	<button id="remove_period">-</button>
-	<br>
-	<label for="period_friendly_name">Friendly Name</label>
-	<input id="period_friendly_name">
-	<br>
-	<label for="period_start">Start Time</label>
-	<input id="period_start">
-	<br>
-	<label for="period_end">End Time</label>
-	<input id="period_end">
-	<br>
-	<label for="period_kind">Period Kind</label>
-	<select id="period_kind">
-		<option value="Class">Class</option>
-		<option value="Lunch">Lunch</option>
-		<option value="Break">Break</option>
-		<option value="AMSupport">AM Support</option>
-		<option value="Passing">Passing (implicit)</option>
-		<option value="BeforeSchool">BeforeSchool (implicit)</option>
-		<option value="AfterSchool">AfterSchool (implicit)</option>
-	</select>
-	<br>
-	<label for="period_class_index">Class Index</label>
-	<input id="period_class_index" type="number">
-	<script src="../editor/controls-section.js"></script>
+	<div id="settings">
+		<button id="pull">Fetch Schedules</button>
+		<button id="push">Apply Changes</button>
+		<button id="update">Force Schedule Reload</button>
+		<span id="controls_error" class="error"></span>
+		<hr>
+		<button id="export">Export settings</button>
+		<br>
+		<label for="import">Import settings:</label>
+		<input type="file" id="import" accept=".json" />
+		<hr>
+		<h2>Manage Schedules</h2>
+		<button id="add_schedule">Create New Blank Schedule</button>
+		<br>
+		<select id="select_schedule">
+		</select>
+		<button id="copy_schedule">Duplicate Selected Schedule</button>
+		<button id="remove_schedule">Delete Selected Schedule</button>
+		<br>
+		<br>
+		<label for="schedule_friendly_name">Schedule Name</label>
+		<input id="schedule_friendly_name" class="schedule_setting">
+		<br>
+		<label for="schedule_code">Schedule Code (internal use)</label>
+		<input id="schedule_code" class="schedule_setting">
+		<br>
+		<label for="schedule_color">Schedule Color</label>
+		<input id="schedule_color" class="schedule_setting" type="color">
+		<br>
+		<label for="schedule_hide">Hide in dropdowns</label>
+		<input id="schedule_hide" class="schedule_setting" type="checkbox">
+		<br>
+		<label for="schedule_regex">Schedule Regex</label>
+		<input id="schedule_regex" class="schedule_setting">
+		<br>
```

Comment:
The layout would make more sense if this section was moved right above the "Manage Periods" section because they are both related. Right now they are split apart by the calendar URLs and the "Typical Week" part.

Reply:
Done